### DISCUSSION TITLE
php

### DISCUSSION THREAD

**Discussion by ruiyuantian** [apache/skywalking#12374] [Q&A]
On Ubuntu, I have installed the PHP agent for my PHP version 7.4. The ecl/skywalking_agent is already installed and is the same as the released version 0.7.0. I have modified the php.ini configuration file, and when I run php -m, I can see the installed extension skywalking_agent. However, SkyWalking is not detecting PHP.

**Comment by heyanlong** [comment]
Hi, what's your running mode on fpm?

**Reply by ruiyuantian** [reply]
dynamic
